panels/model_checks.py

```diff
@@ -45,36 +44,48 @@ def pa_phone(pa):
 
 
 @validation.PanelApplication
-def pa_other(pa):
-    if pa.presentation == c.OTHER and not pa.other_presentation:
+def unavailability(app):
```

Comment:
While I could have simply changed the form to say "available" instead of "unavailable," I figured that will make things very hard for us when it comes to admins viewing panel information later on. So instead we check the "unavailable" field for non-guests and the "available" field for guests.